Lib/test/test_dis.py

```diff
@@ -166,6 +166,19 @@ def bug1333982(x=[]):
        bug1333982.__code__.co_firstlineno + 2,
        bug1333982.__code__.co_firstlineno + 1)
 
+
+def bug42562():
+    pass
+
+
```

Comment:
Better add a comment here, otherwise this is a bit mysterious.
`# Set line number for 'pass' to None`

Reply:
My bad, it's always better to explain magic things)